ledger-api/grpc-definitions/com/daml/ledger/api/v2/participant_offset.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+
+option java_outer_classname = "ParticipantOffsetOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Describes a specific point on the participant. This is a participant local value:
+// changes synchronized via a specific domain should have different participant offsets
+// on different participants.
+//
+// This is also as a unique index of the changes which happened on the virtual shared ledger.
+// Participant offset define an order, which is the same in which order the updates are
+// visible as subscribing to the ``UpdateService``. This ordering is also a fully causal
+// ordering for one specific domain: for two updates synchronized by the same domain, the
+// one with a bigger participant offset happened after than the one with a smaller participant
+// offset. Please note this is not true for updates synchronized by a different domain.
+//
+// The Ledger API endpoints that take offsets allow to specify portions
```

Comment:
I'm not sure what you mean by "portion" here. Is that defined somewhere else in the API?

Reply:
nah, this is the original doc which I kept around